tests/e2e/vsphere_volume_expansion.go

```diff
@@ -1463,17 +1452,6 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(pvclaim).NotTo(gomega.BeNil())
 
-		ginkgo.By("File system resize should not succeed Since SPS service is down. Expect an error")
```

Comment:
Same as above comment . Please add a check for CNS_NEW_SYnc feature enabled or not , then take the decision, whether the volume expansion should complete or not